components/EscapeQuotes.coffee

```diff
@@ -1,30 +1,20 @@
 noflo = require("noflo")
-_ = require("underscore")
 
-class EscapeQuotes extends noflo.Component
+exports.getComponent = ->
+  c = new noflo.Component
+  c.description = "Escape all quotes in a string"
 
-  description: "Escape all quotes in a string"
+  c.inPorts.add 'in',
+    datatype: 'string'
+    description: 'String to escape quotes from'
+  c.outPorts.add 'out',
+    datatype: 'string'
+    description: 'Escaped string'
 
-  constructor: ->
-    @inPorts = new noflo.InPorts
-      in:
-        datatype: 'string'
-        description: 'String to escape quotes from'
-    @outPorts = new noflo.OutPorts
-      out:
-        datatype: 'string'
-        description: 'Escaped string'
+  c.process (input, output) ->
+    return unless input.has 'in'
```

Comment:
This line doesn't have much sense when there's just 1 input port, because that's the only way a process function gets called.